src/DSNInstaller/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+#
+#   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+#   Licensed under the Apache License, Version 2.0 (the "License").
+#   You may not use this file except in compliance with the License.
+#   A copy of the License is located at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#   or in the "license" file accompanying this file. This file is distributed
+#   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+#   express or implied. See the License for the specific language governing
+#   permissions and limitations under the License.
+#
+
+project(dsn_installer)
+
+set(SOURCE_FILES dsn_installer.cpp)
+include_directories(${ESODBC_SRC} ${UT_HELPER} ${IT_HELPER})
```

Comment:
If possible, try not to include UT_HELPER & IT_HELPER. It will add gtest header files to installer.

Reply:
Removed